R/cmapplot.R

```diff
@@ -70,7 +70,7 @@ if (.Platform$OS.type == "windows") {
 } else {
 
   # Assume no Whitney or Calibri on non-Windows (i.e. non-work) computers
-  warning("CMAP theme will default to Arial on non-Windows platforms")
+  message("CMAP theme will default to Arial on non-Windows platforms")
   cmapplot_globals$use_whitney = FALSE
 
   grDevices::X11Fonts(

```

Comment:
@nmpeterson this is unrelated to this PR, but this is my first time taking a close look at cmapplot.R in a while. I am curious if this line (76) or line 109 changes the default behavior of ggplot or other aspects of a user's interaction with their local R installation? I feel that ideally the installation or loading of this package should not change ggplot defaults, if we can avoid it. Thoughts?

Reply:
I think we can discuss it. The reason I didn't have any qualms with it is that I figured people would only be loading this package when they're working on a plot for CMAP, and therefore need those changes to be reflected one way or another. But yeah, probably not "best practices"

Reply:
See line 45 of `theme_cmap.R`. All text in the geom gets changed to the correct font face whenever `theme_cmap()` is called. I would suggest we can therefore delete line 109.

I'm not sure I understand `grDevices::X11Fonts` though.

Reply:
Yes, that makes sense; I'm fine with deleting the final line of `cmapplot.R`.

`grDevices::X11Fonts` has to be used on Mac/Linux instead of `grDevices::windowsFont` to manage the fonts used by the package.